apiclient/__init__.py

```diff
@@ -19,8 +19,6 @@
     sample_tools = None
 from googleapiclient import schema
 
-__version__ = googleapiclient.__version__
```

Comment:
Would you mind getting the version from `pkg_resources` (in case someone is using this version for something)?


```
import pkg_resources

pkg_resources.get_distribution("google-api-python-client").version
```

Reply:
Should I also restore this variable in `googleapiclient.__init__`?

Reply:
I looked through this once more the only place __version__ was used is `model.py`, so I'm fine with deleting it. Thanks for opening this PR!